tests/test_legacy.py

```diff
@@ -1,22 +1,283 @@
 import unittest
-
-import numpy as np
+from pydicom import FileDataset, Dataset
+from pydicom.uid import generate_uid
+from highdicom.legacy import sop
+from datetime import datetime, timedelta
 import pytest
+import enum
 
 
-class LegacyConvertedEnhancedMRImage(unittest.TestCase):
+class Modality(enum.IntEnum):
+    CT = 0
+    MR = 1
+    PET = 2
 
-    def setUp(self):
-        super().setUp()
 
+sop_classes = [('CT', '1.2.840.10008.5.1.4.1.1.2'),
+               ('MR', '1.2.840.10008.5.1.4.1.1.4'),
+               ('PT', '1.2.840.10008.5.1.4.1.1.128')]
 
-class LegacyConvertedEnhancedCTImage(unittest.TestCase):
+
+class TestLegacyConvertedEnhancedImage(unittest.TestCase):
 
     def setUp(self):
         super().setUp()
+        self._modalities = ('CT', 'MR', 'PET')
+        self._ref_dataset_seq_CT = \
+            self.generate_common_dicom_dataset_series(3, Modality.CT)
+        self._ref_dataset_seq_MR = \
+            self.generate_common_dicom_dataset_series(3, Modality.MR)
+        self._ref_dataset_seq_PET = \
+            self.generate_common_dicom_dataset_series(3, Modality.PET)
+        self._output_series_instance_uid = generate_uid()
+        self._output_sop_instance_uid = generate_uid()
+        self._output_series_number = '1'
+        self._output_instance_number = '1'
 
+    def test_output_attributes(self):
+        for m in self._modalities:
+            with self.subTest(m=m):
+                LegacyConvertorClass = \
```

Comment:
I _think_ Converter is the proper spelling.

Reply:
Both [spellings ](https://www.merriam-webster.com/dictionary/converter)look correct. The converter is more popular though. Will change it